### DISCUSSION TITLE
osu!lazer and the future of scorefarming

### DISCUSSION THREAD

**Discussion by WitherFlower** [ppy/osu#16290] [Ideas] [answered]
Throughout the development of osu!lazer, it seems that scorefarming hasn’t been much of a concern to the development team. However, with the eventual conversion to the new scoring system, more and more scorefarmers are getting worried about the future of osu!, and question if they’ll be able to enjoy lazer to its full extent. 

Given the amount of time the scorefarming community invests into the game, we feel like we don’t deserve to be left behind.

This post aims to start the discussion between the scorefarming community and the development team regarding the future of the score ranking.

## What is Scorefarming?

Scorefarming is a playstyle whose primary goal is to gain large amounts of Ranked Score, in order to climb the score ranking.

Scorefarming has been around ever since the very beginning of osu!, and while the playstyle dropped in popularity after the implementation of performance points in 2012, it has seen a resurgence in recent years, with many new players joining the scorefarming community.

## What makes Scorefarming fun?

Because of the current osu! scoring system, scorefarmers are encouraged to play long maps in order to maximise their ranked score gains. 
This has the benefit of rewarding high consistency, thus making a great balance between skill and dedication in the score rankings.

The enjoyment of scorefarming also comes from the fact that you can play anything and still climb the leaderboards. Scorefarming constantly pushes players to play new maps, bringing a lot of variety to the gameplay.

## What is the problem with using standardised scoring for the score ranking?

Using standardised scoring would completely destroy that skill/dedication balance, as it would only favor short and easy maps, which are often less interesting to play and offer less gameplay variety.

_“If every map gives 1 million, why bother playing marathons when any 30 second map gives you the same results ?”_

Furthermore, such a change would undermine the efforts of players who have dedicated thousands of hours to scorefarming, focusing mainly on longer maps.

***

We hope this serves as a good starting point for further discussions, and that future decisions can be taken with more consideration.

&ndash; MrWither83 & Biokax, with love for the scorefarming community.

**Comment by peppy** [comment]
If your main issue is the 1mil score cap, have you tried changing your score display mode to "classic"?

**Reply by WitherFlower** [reply]
I think you misunderstood. This post isn't talking about individual map leaderboards, but the global score ranking (https://osu.ppy.sh/rankings/osu/score)

You said in a previous community meeting that osu!lazer was mainly going to use standardised scoring instead of classic scoring for global score rankings.
![image](https://user-images.githubusercontent.com/52672543/147849011-beceac8e-a772-4a42-a94f-d6b7c92ab0c9.png)

The goal of this post is to explain why doing this will cause more damage than good. 
I hope this clears the misunderstanding.

**Reply by peppy** [reply]
You may want to listen to the meeting rather than read summarised notes as they likely don't give full context.

Either way, my question still stands: Do you have an issue with classic scoring as it is implemented? If so, please focus on that.

**Reply by WitherFlower** [reply]
The main issue I have is that, from what I unserstand, online systems including the global score ranking are going to use standardised instead of classic, effectively damaging the scorefarming community.

**Reply by peppy** [reply]
Again, if it used classic would you have an issue?

**Reply by WitherFlower** [reply]
No that'd be ideal, and it's what I'm trying to explain in the post.
The scorefarming community wants the score ranking to function with classic.

**Reply by Meruru-chan** [reply]
I also think it'd be ideal to use classic scoring totals for "Total Score" statistic as well. Since leveling system depends on it.
If we change it to standardized scoring then leveling up past 100 will become completely ridiculous.
So Basically for Ranked Score / Total Score we should use "Classic Scoring" totals.
For individual map boards toggle between two scoring types. That will be ideal for score farmers and pp players alike.

**Comment by bdach** [comment, marked as answer]
I hope that this is addressed by https://github.com/ppy/osu-queue-score-statistics/pull/157, but time will tell.

**Reply by WitherFlower** [reply]
I think the most important PR is https://github.com/ppy/osu-queue-score-statistics/pull/158, but both of them go together anyway.
Regardless, the issue should be fixed. In the case the scoring still doesn't match expectations for a majority due to the most recent algorithm, I will open a separate issue, but as you said we can't know that until ranked play is out with imported scores and an updated score leaderboard.
